### DISCUSSION TITLE
Co-Pilot Subscription

### DISCUSSION THREAD

**Discussion by midnightmoet** [community/community#136190] [Copilot Conversations] [answered]
GitHub Co-pilot Question

I have clicked to purchase Co-pilot a total of 3 times between this morning and last night but the IDE isn't connecting / recognizing the subscription as being renewed.  I am not sure what exactly is the disconnect but would someone from GitHub check into this as this is a vital tool.

**Comment by master-coco** [comment, marked as answer]
Hey Lisa,
I ran a quick search and found the following steps that I hope helps 


1) Verify Subscription Status:
Go to [GitHub's subscription management page](https://github.com/settings/billing) and check if your subscription is active and associated with the correct account.

2)Sign Out and Sign In:
Try signing out of your IDE and GitHub account, then sign back in. This can sometimes refresh the connection and update the subscription status.

3)Update IDE:
Ensure your IDE is up-to-date. Sometimes, updates include fixes for integration issues with tools like Copilot.

4)Check IDE Configuration:
Confirm that GitHub Copilot is properly configured in your IDE. You might need to reinstall or reconfigure the extension/plugin.

5)Clear Cache:
Clear the cache or local data for your IDE. Corrupted or outdated cache files can sometimes cause connection issues.

6)Contact GitHub Support:
If none of the above steps work, you should reach out directly to [GitHub Support](https://support.github.com/contact). Provide them with details about your subscription attempts and any error messages you’ve encountered.

Hopefully, one of these steps will resolve the issue, and you’ll be back up and running with GitHub Copilot soon.
